### PR TITLE
feat: add validation information PR template

### PR DESCRIPTION
**Pull request by tisidro**
### Description:
Add validation information to PR Template

### Spec:
Designs: [Designs](DESIGN_URL)

See Story: relates to update on - [FSA22V2-215](https://sparkbox.atlassian.net/browse/FSA22V2-215?atlOrigin=eyJpIjoiZjY1OWE4NjYwNzgwNGQzNTgzMTk1MzZlMTBiMjBiM2QiLCJwIjoiaiJ9)

### Validation:
improve validation instructions, specify accessibility testing as this is an accessibility-related site!

* [ ] This PR has code changes, and our linters still pass.
* [ ] This PR affects production code, so it was browser tested (see below).
* [ ] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. In terminal: `npm run dev.`
4. Check [localhost: 3000](http://localhost:3000/) for changes.
5. Run `npm run lint`.
6. Run axe DevTools on affected pages.
7. Confirm all tests pass.

---

#### Browser Testing

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

* [ ] Safari (last 2 major versions)
* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)

**Windows**

* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)
* [ ] Edge (last 6 months)

**Mobile**

* [ ] Safari (last 2 major versions)
* [ ] Chrome on Android (last 6 months)
* [ ] Firefox on Android (last 6 months)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -17,7 +17,10 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: npm run dev.
+4. Check localhost: 3000 for changes.
```
Comment (jkohrt7) [inline .github/pull_request_template.md:21]: Could you remove the space between localhost: and 3000 on this line?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -17,7 +17,10 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: npm run dev.
```
Comment (jkohrt7) [inline .github/pull_request_template.md:20]: This is personal preference, but I like to wrap commands in backticks so it's clear what is and isn't a command, like `npm run dev`.

**Review by jkohrt7 — COMMENTED** [review]
Adding these was a great idea--this'll be super useful going forward. Left a couple nitpicky things for you to consider before I give approval.

File: .github/pull_request_template.md (hunk 2 above)
Comment (tisidro) [reply .github/pull_request_template.md:20]: I like that too! Thank you :)

File: .github/pull_request_template.md (hunk 1 above)
Comment (tisidro) [reply .github/pull_request_template.md:21]: good eye! 👀

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -17,7 +17,11 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: `npm run dev`.
+4. Check localhost:3000 for changes.
```
Comment (empee3) [inline .github/pull_request_template.md:21]: Just a minor nitpick, but consider linking the [localhost:3000](http://localhost:3000/) URL, in the case where someone already has the application running but doesn't have it open in the browser (I do this often).

Comment (tisidro) [reply .github/pull_request_template.md:21]: Great idea, I will do that!

**Review by jkohrt7 — APPROVED** [review]
Changes look good!

**Review by empee3 — APPROVED** [review]

**Review by jkohrt7 — APPROVED** [review]
The link's a good addition!

**Pull request merged**